cmd/flags.go

```diff
@@ -54,6 +54,14 @@ var globalFlags = []cli.Flag{
 		Name:  "insecure",
 		Usage: "Disable SSL certificate verification.",
 	},
+	cli.IntFlag{
+		Name:  "older-than",
+		Usage: "Select objects older than N days",
+	},
+	cli.IntFlag{
+		Name:  "newer-than",
```

Comment:
If these are global flags shouldn't they be not displayed with other commands which don't support this? like `mc share`  `mc ls` `mc events` ... 

We should only add these flags for the relevant commands which do support it.